skimage/transform/_warps.py

```diff
@@ -115,8 +116,9 @@ def resize(image, output_shape, order=None, mode='reflect', cval=0, clip=True,
              "from version 0.19 a ValueError will be raised instead of this "
              "warning.", FutureWarning, stacklevel=2)
 
-    factors = (np.asarray(input_shape, dtype=float) /
-               np.asarray(output_shape, dtype=float))
+    factors = np.divide(input_shape, output_shape)
+    # create copy so input values range stays accessible through image for clip
+    img_in = image.copy()
```

Comment:
I am not sure the copy is necessary here.

Reply:
True @rfezzani, thanks ! Operations later applied on img_in are not *in_place* and thus should not be able to modify image :smiley:.